spec/tags/features/offerwall-dynamic-ui.spec.ts

```diff
@@ -192,4 +192,49 @@ describe('Offerwall dynamic UI', () => {
       id: 'offerwallDynamicUi.complete',
     });
   });
+
+  it('should destroy root and no-logging for irrelevant variant', async () => {
+    const page = await browser.generatePage({
+      variantByFeature: {'offerwall-dynamic-ui': 'test1_exp1'},
+      variables: {
+        'tagsmith.offerwallDynamicUi.headlineText': 'mod headline',
+        'tagsmith.offerwallDynamicUi.bodyText': 'mod body',
+        'tagsmith.offerwallDynamicUi.rewardedAdOptionText':
+          'mod rewarded ad option text',
+        'tagsmith.offerwallDynamicUi.rewardedAdOptionSubtext':
+          'mod rewarded ad option subtext',
+      },
+      forceAbFactor: 0.99,
```

Comment:
QQ: So when forceAbFactor is set to 0.99, does it mean a user is not using offerwall-dynamic-ui feature? (Sorry maybe I still don't understand what is forceAbFactor..)

Reply:
`0.99` maps to `test10_exp3` (see [here](https://github.com/google-marketing-solutions/tagsmith/blob/main/src/snippet.js#L31-L42)) which means the user is not enabled for offerwall-dynamic-ui and isn't in the control variant.